OCExtensions/Classes/ViewController/GeneralViewControllerExtension.swift

```diff
@@ -29,5 +29,26 @@ public extension UIViewController {
         
         return topMost
     }
+ 
+    public func popViewControllers(numberOfPops: Int, animated: Bool = true) {
+        if let navigationController = self as? UINavigationController ?? navigationController {
+            var index = navigationController.viewControllers.count - 1 - numberOfPops
```

Comment:
@cleberhenriques because of index bounds:

`Count starts in 1, and indexes starts in 0`